packages/patternfly-3/patternfly-react-extensions/src/components/VerticalTabs/_mocks_/mockVerticalTabsExample.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import VerticalTabs from '../VerticalTabs';
+
+class MockVerticalTabsExample extends React.Component {
+  state = {
+    activeTabId: 'one'
+  };
+
+  onActivateTab = id => {
+    this.setState({ activeTabId: id });
+  };
+
+  render() {
+    const { activeTabId } = this.state;
+
+    return (
+      <VerticalTabs id="vertical-tabs">
+        <VerticalTabs.Tab
+          id="one"
+          title="Tab One"
+          onActivate={() => this.onActivateTab('one')}
+          active={activeTabId === 'one'}
```

Comment:
Wondering if this `active` tab logic could be pushed down into the `<VerticalTabs>` component?
 `<VerticalTabs id="vertical-tabs" initialActiveTab="one">`
Seems natural that the  `<VerticalTabs>` component keep state of the currently active tab, as opposed to being part of the mockExample, which every app dev will have implement.  The `<VerticalTabs.Tab`'s `onActivate' could then just be a callback function for the app dev. to use or not.

Reply:
We try not to keep state in the components. It's better left to the application since the active tab could change for other reasons.

Reply:
Ok, sounds good.